### DISCUSSION TITLE
Do Models have autocreatetime method?

### DISCUSSION THREAD

**Discussion by acfuns** [SeaQL/sea-orm#877] [Q&A] [answered]
The examples in the documentation are very simple, it can‘t meet the needs of daily development,
eg:
```
#[sea_orm(auto_create_time)]
pub created_at: DataTimeWithTimeZone,

#[sea_orm(auto_update_time)]
pub updated_at: DataTimeWithTimeZone,
```
If I didn't find it, I was sorry.

**Comment by billy1624** [comment, marked as answer]
Hey @acfuns, sorry for the delay. Please check below for the related issues and PRs:
- https://github.com/SeaQL/sea-orm/issues/827
- https://github.com/SeaQL/sea-orm/pull/854

Please join our discussion on GitHub :)
